examples/terraform-aws-lambda-example/README.md

```diff
@@ -0,0 +1,25 @@
+# Terraform Lambda Example
+
+This folder contains a very simple Terraform module to demonstrate how you can use Terratest to deploy a lambda function
```

Comment:
NIT:
```suggestion
This folder contains a Terraform module to demonstrate how you can use Terratest to deploy a lambda function
```

It's usually not a good idea to say "very simple" or even "simple" because that can be subjective.